doc/news/changes/minor/20170726DanielArndt-1

```diff
@@ -0,0 +1,6 @@
+New: The Quadrature class has a new boolean is_tensor_product_flag
```

Comment:
This is actually an implementation detail. I don't think we ought to actually name that flag here. If there is no corresponding getter function, then I'd say just say "The Q class now stores whether the represented...". If there *is* a getter function, name that instead.